src/themes/latex/styles.css

```diff
@@ -4,11 +4,31 @@
 @import '../../extensions/pages/styles.css';
 @import '../../extensions/person/styles.css';
 
-@import 'https://raw.githubusercontent.com/davidrzs/latexcss/9264a50c252d47fbe6d4998dbc5581882d79ab3c/latex.css';
+:--root {
+  @import 'https://raw.githubusercontent.com/davidrzs/latexcss/9264a50c252d47fbe6d4998dbc5581882d79ab3c/latex.css';
 
-:--Article:--root {
```

Comment:
Before merging, I wanted ask if there was a reason you targeted `:--Article:--root` instead of just `:--root` @nokome. Reason for the change being that the `:--Article:--root` CSS selector specificity is higher than the ones being generated by the Theme editor sidebar, so the changed weren't taking hold.

Reply:
No particular reason, just done in haste, and obviously, with some confusion. Will change.